### ISSUE TITLE
Adding style to container

### ISSUE THREAD

**Issue by NgoKnows** [joshwcomeau/react-flip-move#28]
I was wondering if it was possible to style the container of the animated children. For example, I wanted to have the children be in a flexbox layout. Something like this:

```
<FlipMove style={{display: 'flex', justifyContent: 'center'}}>
    { this.renderTopArticles() }
</FlipMove>
```

Otherwise, would the only way of making the elements horizontal would be applying styles to the children themselves?

Thanks for any help.

**Comment by joshwcomeau** [comment]
Yeah, good point. 

I have yet to jump 100% on the inline-styles bandwagon, so I've been using the `className` prop on the container for flexbox and other container styles. That's not an option for everyone, though.

Will add this as a prop.

**Comment by NgoKnows** [comment]
Sounds good! Thanks for the prompt response!

**Comment by joshwcomeau** [comment]
Done! You can use `style` prop as-expected in the just-published v2.1.
